pkg/detectors/launchdarkly/launchdarkly.go

```diff
@@ -77,6 +90,13 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 					defer res.Body.Close()
 					if res.StatusCode >= 200 && res.StatusCode < 300 {
 						s1.Verified = true
+						var tokenResponse tokenResponse
+						if err := json.NewDecoder(res.Body).Decode(&tokenResponse); err == nil && len(tokenResponse.Items) > 0 { // no error in parsing and have atleast one item
+							token := tokenResponse.Items[0]
+							s1.ExtraData["name"] = token.Name
+							s1.ExtraData["role"] = token.Role
+							s1.ExtraData["total_count"] = fmt.Sprintf("%d", tokenResponse.TotalCount)
```

Comment:
What is this a total count of? Can you improve the variable name?

Reply:
LaunchDarkly returns total count of token associated with the detected credentials.